src/Symfony/Component/Form/FormView.php

```diff
@@ -79,11 +79,13 @@ public function isRendered()
     /**
      * Marks the view as rendered.
      *
+     * @param bool $rendered default true
+     *
      * @return $this
      */
-    public function setRendered()
+    public function setRendered(bool $rendered = true)
```

Comment:
Neither the class nor this method is declared `final`, so introducing a new parameter (even an optional one) is a BC break.

Reply:
Agreed. @fabgg We have ways to add new arguments, but it's a bit more involving.